com.playeveryware.eos/CHANGELOG.md

```diff
@@ -4,6 +4,282 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Changelog
 
+## [3.3.2] - 2024-08-27
+
+### Added
+
+- Git Actions
+  - feat: Create check-for-package-difference.yml
+  - feat: Add workflow to auto-update the unity version badge in README.md.
+
+- Plugin General
+  - feature (Sessions): Notifications in subregion
+  - feature (Sessions): Rearranged Functions inside Regions
+  - feature (Sessions): Organizing all functions in to #regions, unsorted otherwise
+  - feature (Sessions): Destroy session when owner destroys their session
+  - feature (Sessions): Refresh UI On Refresh Message
+  - feature (Sessions): Code review feedback implemented
+  - feature (sessions): Reworking Sample, P2P Communication Part
+  - feature(network): Implementing taskNetworkTimeoutSeconds
+  - feat: Make the EOSSettingsWindow aware of which build targets are available, showing platform-specific options for only those that are.
+  - feat: Implement function to enumerate the platforms that can be built against.
+  - feat: Add function to generate a RuntimeConfig struct from the values of an EOSConfig class.
+  - feat: Add RuntimeConfig struct that stores the totality of the configurable properties used by the plugin during runtime. Also add method to EnumUtility that supports decomposing a bitwise-operator combined enum value into it's constituent discrete components.
+  - feat: Add upm layout as upm to asset project
+  - feat: Upgrade targetted version of Unity.
+  - feat: Add ability configure the plugin to not unload the EOS SDK on shutdown.
+  - feature (Sessions): Notifications in subregion
+  - feature (Sessions): Rearranged Functions inside Regions
+  - feature (Sessions): Organizing all functions in to #regions, unsorted otherwise
+  - feature (Sessions): Destroy session when owner destroys their session
+  - feature (Session): Refresh UI On Refresh Message
+  - feature (sessions): Reworking Sample, P2P Communication Part
+
+- Unit Tests
+  - feat: Add unit tests for enum extensions.
+  - feat: Add unit tests of limited scope.
+  - tests(sessions): Create and search for sessions
+  - tests(login): Testing scene for auditing logins
+
+### Changed
+
+- Git and LFS
+  - chore(macos): Store as binary, not text!
+  - chore(macOS): Putting MacOS Binaries in Git LFS
+  - chore: Update .gitattributes
+  - chore: Update Unity version badge to 2021.3.16f1
+
+- Release & Packaging
+  - chore(tool,import) : dispose result of async task to prevent warnings
+  - chore(lib,android) : remove unused android libs
+  - chore(sdk,source) : updated managed source for 1.16.3 hotfix
```

Comment:
update source and update binaries for 1.16.3 can be collapsed into a single 'update eos sdk to 1.16.3' or similar phrasing.